roles/ojs/tasks/main.yml

```diff
@@ -191,3 +191,33 @@
   with_items:
     - 'ja_JP'
   changed_when: '"->" in cp_output["stdout"]'
+
+- name: ojs | clone PUL repository
+  ansible.builtin.git:
+    repo: 'https://github.com/pulibrary/openpublishing.git'
+    dest: /opt/openpublishing
+    version: main
+
+- name: ojs | copy translations to running server
+  ansible.builtin.shell: 'cp -r /opt/openpublishing/plugins/themes/healthSciences/locale/{{ item }} {{ ojs_home }}/plugins/themes/healthSciences/locale/{{ item }}'
+  register: cp_output
+  with_items:
+    - 'ja_JP'
+  changed_when: '"->" in cp_output["stdout"]'
+
+- name: ojs | adjust permissions on PUL repository files
+  file:
+    path: "{{ ojs_home }}/plugins/themes/healthSciences/locale/{{ item }}"
+    state: directory
+    owner: "{{ deploy_user }}"
+    group: www-data
+    mode: 0775
```

Comment:
If you add `recurse: yes` to the parameters for this `file` module task, you should get the correct behavior without the final task in this file. See https://docs.ansible.com/ansible/latest/collections/ansible/builtin/file_module.html#parameter-recurse.